radar-upload-frontend/src/components/Patient/PatientTable/Records.vue

```diff
@@ -34,19 +34,19 @@
       no-action
     >
       <template #activator>
-        <v-list-item-avatar>
-          <!-- <v-progress-circular
+        <!-- <v-list-item-avatar> -->
```

Comment:
just to keep the code clean, can we remove the commented lines? We can add them back when necessary.

Reply:
ok